### PR TITLE
Refactored EventItemModel into separate model and store

### PR DESCRIPTION
**Pull request by Daboo1998**

### CODE REVIEW

File: src/ui-kit/organisms/EventItem/EventItem.tsx [hunk 1]
```diff
@@ -41,6 +41,8 @@ const EventItem: FC<PropsInterface> = ({
 
   useEffect(() => {
     eventStore.init();
+
```
Comment (aminbksi) [inline src/ui-kit/organisms/EventItem/EventItem.tsx:44]: the problem with updating is here, on init either the fetchMagicLink or fetchAttendees has null/undefined field(s) not handled, try to init somewhere else or handle the null/undefined fields

File: src/scenes/widgets/pages/AttendeesWidget/AttendeesWidget.tsx [hunk 2]
```diff
@@ -10,15 +12,46 @@ const AttendeesWidget: FC = () => {
...
+      attendeesListStore.fetchAttendees();
+    },
+    500,
+    [attendeesListStore.query]
+  );
+
   return (
-    <Dialog>
+    <Dialog
+      title="Gathering Name"
```
Comment (aminbksi) [inline src/scenes/widgets/pages/AttendeesWidget/AttendeesWidget.tsx]: translation missing or maybe for title the event title should be here?

File: src/scenes/widgets/pages/AttendeesWidget/AttendeesWidget.styled.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import {rgba} from 'polished';
+import styled from 'styled-components';
+
+import {ProfileWidget} from '../ProfileWidget';
```
Comment (PAKovalev) [inline src/scenes/widgets/pages/AttendeesWidget/AttendeesWidget.styled.ts:4]: Bad idea. Why do we have this relation?

File: src/scenes/widgets/pages/AttendeesWidget/AttendeesWidget.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    },
+    500,
+    [attendeesListStore.query]
+  );
+
+  return (
+    <>
+      <Dialog
+        title="Gathering Name"
+        subtitle={`Attendee List / ${t('counts.attendees', {
```
Comment (PAKovalev) [inline src/scenes/widgets/pages/AttendeesWidget/AttendeesWidget.tsx:29]: i18n

File: src/scenes/widgets/pages/AttendeesWidget/AttendeesWidget.tsx [hunk 5]
```diff
@@ -23,34 +23,50 @@ const AttendeesWidget: FC = () => {
...
-              <Avatar avatarSrc={attendee.avatarSrc} size="small" />
-              <Text size="s" text={attendee.name} />
-            </styled.Item>
-          ))}
-        </styled.List>
-      </styled.Container>
-    </Dialog>
+    <>
+      <Dialog
+        title="Gathering Name"
```
Comment (aminbksi) [inline src/scenes/widgets/pages/AttendeesWidget/AttendeesWidget.tsx:28]: same here

**Review by PAKovalev — CHANGES_REQUESTED** [review]

File: src/scenes/widgets/pages/AttendeesWidget/AttendeesWidget.styled.ts (hunk 3 above)
Comment (Daboo1998) [reply src/scenes/widgets/pages/AttendeesWidget/AttendeesWidget.styled.ts:4]: I need to style Profile Widget to show next to the AttendeesWidget
I can change import

**Review by aminbksi — APPROVED** [review]

**Review by PAKovalev — APPROVED** [review]

**Pull request merged**